src/main/java/fi/okm/jod/yksilo/config/SecurityConfig.java

```diff
@@ -61,11 +62,8 @@ public SecurityFilterChain securityFilterChain(
             request.getSession(false) == null
                 || SecurityContextHolder.getContext().getAuthentication() == null;
 
-    // for development, to be removed
-    // makes it possible to test the API locally using Swagger UI
-    if (!env.matchesProfiles("cloud")
-        && Boolean.TRUE.equals(
-            env.getProperty("springdoc.swagger-ui.enabled", Boolean.class, false))) {
+    var isLocalProfileActive = Arrays.stream(env.getActiveProfiles()).anyMatch("local"::equals);
```

Comment:
I suppose `env.matchesProfiles("local")` would also work. And maybe it would be good to keep the swagger-ui.enabled check also.